### PR TITLE
Carousel

### PR DESCRIPTION
**Pull request by kanotten**
added slider/carousel feature to index, also added png´s to assets folder. 

pictures need to be changed, and needs to be linked to a blog post. 
added its own css file for the carousel.

### CODE REVIEW

File: .DS_Store
Comment (itacud95) [inline .DS_Store:1]: Dont check in this

File: javascript/script.js
Comment (itacud95) [inline javascript/script.js:1]: naming convention would be carousel.js

Comment (itacud95) [reply javascript/script.js:1]: my bad, you did not add a carousel.html, so this is fine

File: .DS_Store
Comment (kanotten) [reply .DS_Store:1]: thanks, will remove

**Comment by itacud95** [comment]
lgtm

**Pull request merged**